cScripts/CavFnc/functions/logistics/fn_doStarterCrateSupplies.sqf

```diff
@@ -159,7 +159,7 @@ if (_companySelector in _alpha or _alwaysAvalible) then {
         //================== AMMO ==================\\
         _crate addMagazineCargoGlobal ["rhs_mag_30Rnd_556x45_M855A1_Stanag",250];
         _crate addMagazineCargoGlobal ["rhs_mag_30Rnd_556x45_M855A1_Stanag_Tracer_Red",50];
-        _crate addMagazineCargoGlobal ["rhs_200rnd_556x45_M_SAW",250];
+        _crate addMagazineCargoGlobal ["rhsusf_200rnd_556x45_mixed_box",250];
```

Comment:
Self note: this needs to be done in the crate remake due to merge request later on.